Products/Zuul/search/routers.py

```diff
@@ -80,10 +81,17 @@ def _getLoggedinUserId(self):
         securityManager = getSecurityManager()
         return securityManager.getUser()._login
 
+    def _sanitizeQuery(self, query):
+        query = re.sub('[*]', '', query)
```

Comment:
We need to reduce multiple wildcards to a single wildcard, but not completely eliminate them.